superset/config.py

```diff
@@ -236,7 +236,6 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 
 # Specify the App icon
 APP_ICON = "/static/assets/images/superset-logo-horiz.png"
-APP_ICON_WIDTH = 126
```

Comment:
Should we add something to `UPDATING.MD` stating that this configuration was removed?

Reply:
Right! Added to the breaking changes. Thanks @michael-s-molina 

@rusackas let me know your thoughts about communicating this breaking change to the wider community before merging.

Reply:
@rusackas I think the configurability brings a bit of complexity to it. If there is a strong signal from the community that they need the `APP_ICON_WIDTH`, we should probably add another config flag, such as `USE_APP_ICON_FORCED_WIDTH = Boolean`, and then behave consistently with that when True. When False just fallback to the implementation of this PR.